src/cascadia/TerminalSettingsModel/ActionArgs.cpp

```diff
@@ -582,4 +583,9 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         }
         return RS_(L"ResetWindowNameCommandKey");
     }
+
+    winrt::hstring GlobalSummonArgs::GenerateName() const
+    {
+        return L"";

```

Comment:
Shouldn't the name be something like `Summon the "<name>" window`? And if the name is `_quake`, we make it something like `Summon Quake window`?

Reply:
You know, I left these blank because I thought "these don't make any sense in the cmdpal". But we also use this logic for the Actions page, so I'm just gonna do it after all